searchlib/src/vespa/searchlib/test/features/distance_closeness_fixture.cpp

```diff
@@ -24,16 +66,47 @@ DistanceClosenessFixture::DistanceClosenessFixture(size_t fooCnt, size_t barCnt,
         SimpleTermData term;
         term.setUniqueId(fooCnt + i + 1);
         term.addField(fieldId).setHandle(barHandles.back());
+        if (!query_tensor.empty()) {
+            term.set_query_tensor_name("qbar");
+        }
         queryEnv.getTerms().push_back(term);
     }
+    if (!query_tensor.empty()) {
+        attr_tensor = create_attribute_tensor("bar", "tensor(x[2])", docid_limit);
```

Comment:
Consider renaming variable to `tensor_attr`

Reply:
Done